conf/route_control.py

```diff
@@ -296,6 +297,17 @@ def probe_addr(item, src_mac):
     arpcache[item.neighbor_ip] = item
     print('Adding entry {} in arp probe table'.format(item))
 
+    try:
+        ipb = ipaddress.ip_address(item.neighbor_ip)
+        print("The IP address {} is valid.".format(ipb))
```

Comment:
Why do you need this `print` if you are also printing it inside the `if` and `else` statements?
The output for a valid IPv4 address is like this (which seems redundant):
```python
The IP address 6.6.6.6 is valid.
The IP address 6.6.6.6 is valid ipv4 address
```
or for an IPv6 address:
```python
The IP address 2001:4860:4860::8888 is valid.
The IP address 2001:4860:4860::8888 is valid ipv6 address. Ignore
```

```suggestion
```